repo/packages/S/scale/2/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/ngageoint/scale.git",
+  "maintainer": "scale@nga.mil",
+  "website": "https://ngageoint.github.io/scale",
+  "name": "scale",
+  "tags": [
+    "bigdata",
+    "batch",
+    "analytics",
+    "remotesensing",
+    "algorithms",
+    "geospatial"
+  ],
+  "framework": true,
+  "version": "4.4.1-1.0.0",
+  "preInstallNotes": "We recommend a minimum of three nodes with at least 4 CPU and 6GB of RAM available for the Scale services and running Scale jobs. By default, Elasticsearch package *must* be running within your DCOS cluster. If you wish to use an externally hosted Elasticsearch cluster, specify one or more of the nodes in comma delimited format in the SCALE_ELASTICSEARCH_URLS variable. For quick-start purposes, Scale is bootstrapped with a Postgres database. This should *NEVER* be used for production purposes as it offers no underlying storage persistence. It can be replaced with an externally hosted Postgres by setting DB_HOST and associated settings appropriately. If you are running DCOS 1.8 Enterprise Edition or higher, you will need to set the DCOS_OAUTH_TOKEN in the DCOS section of the Advanced Settings. This value can be found within the dcos.toml file under in the dcos_acs_token value on a system with an authenticated DCOS CLI.\n\nTutorial documentation for Scale can be found here: https://github.com/dcos/examples/tree/master/scale/1.8",
```

Comment:
Add `This DC/OS Service is currently in preview.` at the beginning of `preInstallNotes`